docusaurus-docs/jupyterlab-conda-store/introduction.md

```diff
@@ -1,7 +1,71 @@
 ---
-sidebar_position: 1
 title: Introduction
 description: Introduction to JupyterLab Extension.
 ---
 
 # conda-store JupyterLab extension
+
+A extension to use the [conda-store UI][conda-store-ui] - a React-based frontend for conda-store, within JupyterLab.
+
+## Install 📦
+
+1. Pre-requisites: `conda-store-server`, JupyterLab `>=` 3.0 and `<=` 4.0, and Python `>=` 3.8 installed.
+
+2. Install the extension:
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+
+<TabItem value="conda" label="conda" default>
+
+```bash
+conda install -c conda-forge jupyter-lab-conda-store
+```
+
+</TabItem>
+
+<TabItem value="pip" label="pip" default>
+
+```bash
+pip install jupyterlab-conda-store
+```
+
+</TabItem>
+
+</Tabs>
+
+3. Start JupyterLab:
+
+```bash
+jupyter lab
+```
+
+4. (Optional) Uninstall the extension
+
+<Tabs>
+<TabItem value="conda" label="conda" default>
+
+```bash
+conda uninstall jupyter-lab-conda-store
+```
+</TabItem>
+<TabItem value="pip" label="pip" default>
+
+```bash
+pip uninstall jupyterlab-conda-store
+```
+</TabItem>
+</Tabs>
+
+## Usage
+
+In the JupyterLab window, click on `conda-store` menu-bar item to open the UI in a new window within JupyterLab.
```

Comment:
Would be nice to have a screen shot here of opening the conda-store from within JupyterLab

Reply:
Indeed, added!